packages/sui-ssr/server/ssr/index.js

```diff
@@ -36,8 +36,16 @@ const useLegacyContext =
     ? ssrConfig.useLegacyContext
     : true
 
+const setContentType = res => {
```

Comment:
Maybe a Hook is a better place to set up the content-type of your req.

Reply:
We've tried with a hook but seems that this is executed after any hook and so the Content-Type always gets override